src/settings.js

```diff
@@ -1543,6 +1543,12 @@ var ASAN_SHADOW_SIZE = 33554432;
 // This is enabled automatically when using -g4 with sanitizers.
 var LOAD_SOURCE_MAP = 0;
 
+// Whether we should use the offset converter.
+// This is needed for older versions of v8 that does not give the hex module offset
```

Comment:
It would be good to say which version specifically so we know when we can remove this.

Reply:
Well, we need the offset converter for other purposes, but at least we can remove the comment that says it's needed for older versions of v8. The comment is helpful anyways though.